### DISCUSSION TITLE
Move all bot issues and pull requests into a separate tab

### DISCUSSION THREAD

**Discussion by nickserv** [community/community#4520] [Projects and Issues]
I help maintain Node.js packages in large organizations, and my notifications are frequently filled with automated dependency updates (Dependabot) and documentation updates that do not affect me. Since these automated bot issues and pull requests are a lower priority than replying to human users and contributors, I'd like to hide them from my issues and pull requests, including the ability to separate opt in/out of notifications for them. A new "Bot Updates" tab could automatically categorize bot issues and pull requests and clean up the review interface. Additionally since many bot updates are trivial (for example I always want to merge a dependency update if it's passing CI), there should be a way to opt in entire categories of updates for automatic merging/closing based on required checks (similarly to the existing UI for required checks and automated merging of human pull requests).

**Comment by erikbjoern** [comment]
I agree on the usefulness of this functionality. I would also like to be able to adjust the default search parameters for the Pull Requests view - either by plainly having a setting for defaults, or an option to set up various custom views with pre-defined parameters

**Comment by niolenki** [comment]
In addition to default filters, could be nice to have some different filter configurations levels:
- custom organization wide filters
- custom repository wide filters
- custom user filters

**Comment by trallnag** [comment]
Yup. While I enjoy automations like Dependabot, I dislike how automated pull requests pollute the pull request tab in the GitHub UI.
